docs/usage.md

```diff
@@ -347,17 +352,28 @@ Minimum mapping quality. Reads with lower quality are discarded. Default: 10
 
 ### Digestion Hi-C
 
+#### `--digestion`
```

Comment:
That's a good point. I do not know exactly how the other pipelines are using this `usage` file. I guess I could update it to be less redundant with the `json`. I propose to think about that for a next version if it's ok for you because I would like to release this one quickly. But I'll open an issue to keep track of it